### ISSUE TITLE
Code Editor 2: No autocomplete at all when typing inside a static method

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#7766]
### Description
When typing code inside a static method, no autocomplete appears at all, not even the basic things that normally appear in autocomplete regardless of context.

https://github.com/user-attachments/assets/fc58cbd3-fec6-44c3-a9d4-f6ab353976dc

If the method is not static, then autocomplete sometimes works, but won't work if it's inside or below some sort of statement and is not the last line of the method:

https://github.com/user-attachments/assets/120f969f-55ce-4591-ba72-60b4e702f824

### Steps To Reproduce
```
function test() constructor {
	static draw = function() {
		//type anything here
	}
}
```

```
function test() constructor {
	draw = function() {
		if thing {
			//type anything here
		}
		//autocomplete works here
	}
}
function test() constructor {
	draw = function() {
		if thing {
			//autocomplete does not work here
		}
		//autocomplete does not work here
                if otherThing {
                     //autocomplete does not work here
                }
                //autocomplete works here
	}
}
```
### Which version of GameMaker are you reporting this issue for?
IDE v2024.1100.0.626 Runtime v2024.1100.0.652

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22631.0

[151ecf9a-6246-4530-aec0-abde95c00cc1](https://api.gamemaker.io/api/github/downloads/151ecf9a-6246-4530-aec0-abde95c00cc1)

**Comment by zreedy** [comment]
Fixed

**Comment by Emc1923** [comment]
Verified fixed as of  IDE v2024.1100.0.666  Runtime v2024.1100.0.692
